### PR TITLE
Several changes in query panel

### PR DESCRIPTION
**Pull request by bowenlan-amzn**
1. Correct default for metric/group by
   Implemented by saving selectedOption in its own component state, kind of misuse of FormikComboBox
2. Hide add button dynamically
   

https://user-images.githubusercontent.com/62091230/130013064-d3bb1913-0103-437b-a55f-11e3d6dec861.mp4

3. Fix data filter showing
   This where filter now using the same structure as metrics and groupBy: badge popover and add button.
4. Run query move to DefineMonitor componentDidUpdate, remove Run button.
5. Showing Alerting/Alerts on Dashboard breadcrumb
5. Using biggest end/acknowledge time as `Trigger last updated` time
6. Giving title to graph Content Panel

### CODE REVIEW

File: public/pages/CreateMonitor/containers/DefineMonitor/utils/mappings.js [hunk 1]
```diff
@@ -43,6 +43,8 @@ export function getFieldsFromProperties(properties, dataTypes, path) {
 }
 
 export function getTypeFromMappings(mappings, dataTypes, path = '') {
```
Comment (leeyun-amzn) [inline public/pages/CreateMonitor/containers/DefineMonitor/utils/mappings.js:45]: Do we want to remove these 2 comments?

File: public/pages/CreateMonitor/components/MonitorExpressions/expressions/MetricPopover.js [hunk 2]
```diff
@@ -35,73 +35,98 @@ export default function MetricPopover(
...
+            }}
+          >
+            Cancel
+          </EuiButtonEmpty>
         </EuiFlexItem>
         <EuiFlexItem>
           <EuiButton
             fill
             onClick={() => {
+              console.log('save metric');
```
Comment (leeyun-amzn) [inline public/pages/CreateMonitor/components/MonitorExpressions/expressions/MetricPopover.js:126]: Do we want to remove this?

File: public/pages/CreateMonitor/containers/DefineMonitor/utils/mappings.js (hunk 1 above)
Comment (bowenlan-amzn) [reply public/pages/CreateMonitor/containers/DefineMonitor/utils/mappings.js:45]: Want to keep here for people to understand structure of index mapping easily

File: public/pages/CreateMonitor/components/MonitorExpressions/expressions/MetricPopover.js (hunk 2 above)
Comment (bowenlan-amzn) [reply public/pages/CreateMonitor/components/MonitorExpressions/expressions/MetricPopover.js:126]: Removed

**Review by leeyun-amzn — APPROVED** [review]

**Pull request merged**